aviary/mission/gasp_based/ode/base_ode.py

```diff
@@ -214,3 +217,36 @@ def AddThrottleControl(
                 prop_group,
                 promotes=['*']
             )
+
+    def add_flight_conditions(self, nn, input_speed_type=SpeedType.TAS):
+        self.add_subsystem(
+            "flight_conditions",
+            FlightConditions(num_nodes=nn, input_speed_type=input_speed_type),
+            promotes_inputs=["rho", Dynamic.Mission.SPEED_OF_SOUND,
+                             ("TAS", Dynamic.Mission.VELOCITY)],
+            promotes_outputs=[Dynamic.Mission.DYNAMIC_PRESSURE,
+                              Dynamic.Mission.MACH, "EAS"],
```

Comment:
I think you need to modify the inputs and outputs list based on input_speed_type

Reply:
Good call, modified. There are no cases where this came up but it'll be good to have it correct.